grails-app/services/de/iteratec/osm/measurement/environment/BrowserService.groovy

```diff
@@ -27,33 +28,37 @@ class BrowserService {
         return Collections.unmodifiableSet(result)
     }
 
-    List<Browser> findAllByNameOrAlias(List<String> browserNameOrAlias) {
+    List<Browser> findAllByName(List<String> browserNames) {
         List<Browser> result = []
-        browserNameOrAlias.each {
+        browserNames.each {
             result << findByNameOrAlias(it)
         }
         return result
     }
 
-    Browser findByNameOrAlias(String browserNameOrAlias) {
-        Browser ret = Browser.findByName(browserNameOrAlias)
-        if (ret == null) {
-            return findByAlias(browserNameOrAlias)
+    Browser findByNameOrAlias(String browserName) {
+        Browser browser = Browser.findByName(browserName)
+        if (browser == null) {
+            return findByAlias(browserName)
         } else {
-            return ret
+            return browser
+        }
+    }
+
+    private Browser findByAlias(String browserName) {
```

Comment:
I guess this whole method could be as simple as `BrowserAlias.findByAlias(browserName)?.browser`, or am I missing something?